pkg/osbuild/kickstart_stage.go

```diff
@@ -149,6 +149,70 @@ func NewKickstartStageOptions(
 		users = usersOptions.Users
 	}
 
+	var rootpw *RootPasswordOptions
+	for name, user := range users {
+		// User options for the root user have no effect in kickstart. In other
```

Comment:
I wonder if extracting this into a helper would be slightly easier to read, then the helper name could convey that this is exclusively about adjusting/validating the root user. That would have helped me I think, from a casual reading it was not obvious to me that the loop is only dealing with root and the "break" at the end also made me pause and wonder (but of course if its all about root it makes perfect sense).

Reply:
Indeed, it's much more readable as a function.